src/api_pull/clear_incoming_data_dir.R

```diff
@@ -0,0 +1,42 @@
+clear_incoming_data_dir <- function(incoming_dir, archive_dir) {
+  # Check if the previous step in the targets pipeline ran
+  # (This check can be omitted if not needed)
+
+  # List files in the incoming and archive directories
+  incoming_files <- list.files(incoming_dir, full.names = FALSE)
+  archive_files <- list.files(archive_dir, full.names = FALSE)
+
+  # Find files that are not already in the archive directory
+  files_to_copy <- setdiff(incoming_files, archive_files)
+
+  # Copy only the files that are not already in the archive directory
+  if (length(files_to_copy) > 0) {
+    for (file in files_to_copy) {
+      full_file_name <- file.path(incoming_dir, file)
+      file.copy(full_file_name, archive_dir)
+      print(paste0(file, " has been moved to archive API data folder."))
+    }
+    print("Files have been copied from the incoming directory to the archive directory.")
+  } else {
+    print("All files are already present in the archive directory. Nothing to copy.")
+  }
+
```

Comment:
```suggestion
Sys.sleep(5)

# refresh archive_files now that the incoming_files have been copied over
archive_files <- list.files(archive_dir, full.names = FALSE)
```

... or some other increment of time - just to buy a little time for the transfer to have completed. Not sure what the file moving situation will look like once we move it to FC's system...

And, we need to refresh the list of names in the archive_files or we will keep getting the message "Not all files from the incoming directory have been successfully copied to the archive directory."